appveyor.yml

```diff
@@ -13,28 +13,16 @@ environment:
 
   # We set a default Python version for the miniconda that is to be installed. This can be
   # overridden in the matrix definition where appropriate.
+  CONDA_PY: "27"
   BINSTAR_TOKEN:
     # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
     secure: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
 
   matrix:
-    - TARGET_ARCH: x64
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
     - TARGET_ARCH: x86
-      CONDA_PY: 35
 
-    - TARGET_ARCH: x86
-      CONDA_PY: 27
+    - TARGET_ARCH: x64
```

Comment:
I just noted that the other `vcs` are missing in the channel. @msarahan can you comment if that was by design or should we fix this?